crapssim/player.py

```diff
@@ -19,59 +29,70 @@ class Player(object):
         Sum of bet value for the player
     """
 
-    def __init__(self, bankroll, bet_strategy=None, name="Player"):
-        self.bankroll = bankroll
-        self.bet_strategy = bet_strategy
-        self.name = name
-        self.bets_on_table = []
-        self.total_bet_amount = 0
-        # TODO: initial betting strategy
+    def __init__(self, bankroll: float,
+                 bet_strategy: STRATEGY_TYPE = passline,

```

Comment:
I'll go ahead and let @skent259 weigh in on that one as that TODO was his, but since I was in there anyways I thought I would add it. I'm fine with keeping it as None. If all players have no strategy do we want the table to still run, or would we be better off either erroring out or making table.should_keep_rolling False?

Reply:
Setting `table.should_keep_rolling` to False sounds like the logical thing to do here.

Reply:
Let's have the initial betting strategy as pass line.  As long as it's documented this shouldn't cause confusion to the end user.  This keeps the code base simpler as well